tools/check_bugrefs

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+import configparser
+import os.path
+import requests
+import sys
+
+config = configparser.ConfigParser()
+config.read(os.path.expanduser('/home/okurz/.oscrc'))
```

Comment:
ah, ok. I better include my password directly then ;-) Sorry, there is a reason I used 'expanduser'. I just didn't make proper use of it.